x/wasm/client/cli/query.go

```diff
@@ -35,10 +36,31 @@ func GetQueryCmd() *cobra.Command {
 		GetCmdGetContractHistory(),
 		GetCmdGetContractState(),
 		GetCmdListPinnedCode(),
+		GetCmdLibVersion(),
```

Comment:
I was expecting this at the command root level, too but I can see that it is easier to maintain this way 👍